pkg/api/controllers/transaction_controller_test.go

```diff
@@ -369,7 +370,11 @@ func TestTooManyClient(t *testing.T) {
 				for i := 0; i < pgtesting.MaxConnections; i++ {
 					tx, err := store.(*sqlstorage.Store).Schema().BeginTx(context.Background(), &sql.TxOptions{})
 					assert.NoError(t, err)
-					defer tx.Rollback()
```

Comment:
The error in the case of rollback must be handled silently since the transaction can have been committed later in the function.
It follows the guidelines in https://go.dev/doc/database/execute-transactions